src/operation/emulation_model.jl

```diff
@@ -463,173 +453,109 @@ Default solve method for an Emulation model used inside of a Simulation. Solves
 - `model::EmulationModel`: Emulation model
 - `start_time::Dates.DateTime`: Initial Time of the simulation step in Simulation time.
 - `store::SimulationStore`: Simulation output store
-
-# Accepted Key Words
-- `exports`: realtime export of output. Use wisely, it can have negative impacts in the simulation times
 """
 function run!(
     step::Int,
     model::EmulationModel{<:EmulationProblem},
     start_time::Dates.DateTime,
-    store::SimulationStore;
-    exports = nothing,
+    store::SimulationStore,
 )
     # Initialize the InMemorySimulationStore
     solve_status = run!(model)
     if solve_status == RunStatus.SUCCESSFUL
-        write_problem_results!(step, model, start_time, store, exports)
+        write_problem_results!(step, model, start_time, store)
         advance_execution_count!(model)
     end
 
     return solve_status
 end
 
-function write_results!(model::EmulationModel)
-    # substitute with getters
-    store = model.store
-    container = get_container(model)
+list_aux_variable_keys(x::EmulationModel) =
```

Comment:
the list methods should be moved to `operation_model_interface` since they should potentially work on DecisionModels too